core/meta/datastructures/v1/download.go

```diff
@@ -7,5 +7,6 @@ type DownloadInfo struct {
 	FileName    string // can be empty
 	Target      string // type of artifact to download
 	Name        string // name of artifact to download
+	ID          string // ID of artifact to download (relevant only for controls)
```

Comment:
The `name` means identifier.
There is no need to add a new field.
If this is confusing, replace the `name` filed with `ID`/`identifier`

Reply:
Done